config/default.yml

```diff
@@ -154,6 +154,12 @@ PreCommit:
       - '**/Makefile'
       - '.gitmodules'
 
+  HtmlTidy:
```

Comment:
In general, tidy [identifies and corrects markup errors](http://www.html-tidy.org/documentation/). Given the `-errors` flag, it foregoes the corrections and just outputs the errors.

It's not related to `W3cHtml`, though their rulesets likely overlap. It also doesn't make any remote requests, which is a plus.